libs/coin-modules/coin-evm/src/__tests__/coin-tester/scenarios/ethereum.ts

```diff
@@ -0,0 +1,252 @@
+import Eth from "@ledgerhq/hw-app-eth";
+import { BigNumber } from "bignumber.js";
+import { ethers, providers } from "ethers";
+import { killSpeculos, spawnSpeculos } from "@ledgerhq/coin-tester/signers/speculos";
+import { encodeTokenAccountId } from "@ledgerhq/coin-framework/account/index";
+import { Scenario, ScenarioTransaction } from "@ledgerhq/coin-tester/main";
+import { getCryptoCurrencyById } from "@ledgerhq/cryptoassets/currencies";
+import { buildAccountBridge, buildCurrencyBridge } from "../../../bridge/js";
+import { makeAccount } from "../../fixtures/common.fixtures";
+import { EvmNftTransaction, Transaction as EvmTransaction } from "../../../types";
+import resolver from "../../../hw-getAddress";
+import { setCoinConfig } from "../../../config";
+import {
+  ethereum,
+  ERC20Interface,
+  USDC_ON_ETHEREUM,
+  ERC721Interface,
+  ERC1155Interface,
+} from "../helpers";
+import { clearExplorerAppendix, getLogs, setBlock } from "../indexer";
+import { killAnvil, spawnAnvil } from "../anvil";
+
+const scenarioSendTransaction: ScenarioTransaction<EvmTransaction> = {
+  name: "Send ethereum",
+  amount: new BigNumber(100),
+  recipient: "0x6bfD74C0996F269Bcece59191EFf667b3dFD73b9",
+  expect: account => {
+    const { operations } = account;
+    const sendOperation = operations.find(operation => operation.transactionSequenceNumber === 0);
+    expect(sendOperation?.transactionRaw?.amount === "100").toBeDefined();
+  },
+};
+
+// use function createTransaction
+const scenarioUSDCTransaction: ScenarioTransaction<EvmTransaction> = {
+  name: "Send USDC",
+  amount: new BigNumber(80),
+  recipient: "0x6bfD74C0996F269Bcece59191EFf667b3dFD73b9",
+  subAccountId: encodeTokenAccountId(
+    "js:2:ethereum:0x3313797c7B45F34c56Bdedc0179992A4d435AF25",
+    USDC_ON_ETHEREUM,
+  ),
+  expect: account => {
+    const { operations } = account;
+    const sendOperation = operations.find(operation => operation.transactionSequenceNumber === 1);
+    expect(sendOperation?.transactionRaw?.amount).toBe("80");
+  },
```

Comment:
Same here but even easier:
- TokenAccount balance is now 100 - 80
- Account has now 6 ops (if my initial assumption of account having 3 ops after setup)
- Latest op should have type FEES & 1 subOperation of type OUT and value of 80^6 & contract is USDC's address